docs/CHANGELOG.asciidoc

```diff
@@ -60,6 +61,322 @@ Review important information about the {kib} 8.x releases.
+* Fixes quick create geo job created by ID ({kibana-pull}177691[#177691]).
+* AIOps: Fixes incomplete edge buckets for change point detection ({kibana-pull}177579[#177579]).
+* AIOps: Fixes grouping for fields with large arrays ({kibana-pull}177438[#177438]).
+* Fixes Single Metric Viewer's zoom settings in URL are not restored if URL specifies a forecast ID ({kibana-pull}176969[#176969]).
+* Adds delay to deletion modal to avoid flickering ({kibana-pull}176424[#176424]).
+* Fixes Single Metric Viewer not showing chart for metric functions and mismatch function in tooltip ({kibana-pull}176354[#176354]).
+* Fixes multi-match query overriding filters in Data Visualizer and Data Drift ({kibana-pull}176347[#176347]).
+* Fixes only enabling apply button in anomaly detection datafeed chart if changes have been made ({kibana-pull}174425[#174425]).
+Management::
+* Fixes editing a rollup data view ({kibana-pull}177446[#177446]).
+* Fixes showing previously selected no time field setting ({kibana-pull}177221[#177221]).
+* Fixes package showing 'Needs authorization' warning even after transform assets were authorized successfully ({kibana-pull}176647[#176647]).
+* Removes the polling interval to reload indices in the background ({kibana-pull}174681[#174681]).
+* Fixes keeping the filters value in the URL for the indices list ({kibana-pull}174515[#174515]).
+* Fixes categorizing fields as empty that never had a value in matching indices ({kibana-pull}174063[#174063]).
+* Fixes the badge for managed data streams in Index Management ({kibana-pull}173408[#173408]).
+* Some input fields are now disabled when editing managed repositories in Snapshot & Restore ({kibana-pull}173137[#173137]).
+* Input fields to change snapshot name and repository are now disabled when editing managed SLM policies in Snapshot & Restore ({kibana-pull}172291[#172291]).
+Maps::
+* Fixes Request URL Too Long (414) with heatmap layer when data view has larger number of date fields ({kibana-pull}177900[#177900]).
+* Fixes the maps application breaking if you open a map with layers or sources that do not exist (#176419).
+Observability::
+* SLOs: Does not display group by cardinality when group by is not selected ({kibana-pull}178133[#178133]).
+* Fixes bug in inventory rule for Inbound and Outbound traffic threshold (both preview and executor) ({kibana-pull}177997[#177997]).
+* Fixes more lenient parsing of suggestion scores ({kibana-pull}177898[#177898]).
+* Fixes refreshing the conversations list on conversation update ({kibana-pull}177897[#177897]).
+* Fixes SLO details path is broken when `instanceId` contains a forward slash ({kibana-pull}177843[#177843]).
+* Prevents users from picking date fields for the group-by selector ({kibana-pull}177830[#177830]).
+* Fixes undefined issue cased due row check missing ({kibana-pull}177293[#177293]).
+* Reverts changes to `useLicense` hook ({kibana-pull}174372[#174372]).
```

Comment:
this just caught my attention because of the title, not our team, but a revert title points to something that maybe didn't make it into production, needs to be double checked, but this could be the case, since this was reverted: https://github.com/elastic/kibana/pull/174031